packages/synthetics-sdk-broken-links/src/broken_links.ts

```diff
@@ -88,11 +88,13 @@ export async function runBrokenLinks(
   let browser: Browser;
   try {
     const options = processOptions(inputOptions);
+    const adjusted_synthetic_timeout_millis =
+      options.total_synthetic_timeout_millis! - 7000;
```

Comment:
Any particular reasoning on the 7 sec? Seems like before it was 10 sec?

Reply:
Made it 7 rather than 10 because that seemed like an unnecessarily long amount of time difference + the mocha offset is 7 secs so thought it would be good to keep consistent.